<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User1</class>
 <widget class="QWidget" name="User1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-laugh"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Change User</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CURRENT DATE :</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>View Friends</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="user_2_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="icon_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="user_3_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>View Liked Pages</string>
   </property>
  </widget>
  <widget class="QToolButton" name="user_1_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="icon_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>View Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>View Timeline</string>
   </property>
  </widget>
  <widget class="QToolButton" name="user_4_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bday">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>View All Posts</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Del Memories</string>
   </property>
  </widget>
  <zorder>icon_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>icon_1</zorder>
  <zorder>name</zorder>
  <zorder>id</zorder>
  <zorder>bd</zorder>
  <zorder>label_4</zorder>
  <zorder>date_label</zorder>
  <zorder>toolButton</zorder>
  <zorder>icon_3</zorder>
  <zorder>user_2_btn</zorder>
  <zorder>icon_2</zorder>
  <zorder>user_3_btn</zorder>
  <zorder>toolButton_4</zorder>
  <zorder>user_1_btn</zorder>
  <zorder>icon_4</zorder>
  <zorder>toolButton_2</zorder>
  <zorder>toolButton_3</zorder>
  <zorder>user_4_btn</zorder>
  <zorder>bday</zorder>
  <zorder>toolButton_5</zorder>
  <zorder>toolButton_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
